iac-extensions/kubernetes/src/main/java/org/sonar/iac/kubernetes/model/ProjectResource.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * SonarQube IaC Plugin
+ * Copyright (C) 2021-2024 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.iac.kubernetes.model;
```

Comment:
Not sure if it should be placed somewhere deeper in the hierarchy

Reply:
IMO it is ok to place it here. I'm not super happy with the `model` package name as it is too general world. In sonar-php the `symbol` package is used for a similar purpose, but we don't have symbols here.
Maybe `project` or `shared` or `xfile` or `crossfile` will be a better name here? (I don't have a strong opinion here).

Reply:
I'll stick with the `model` then